package.json

```diff
@@ -37,5 +37,8 @@
     "multidep": "^2.0.0",
     "prettier": "^1.16.4",
     "rsvp": "^4.8.4"
+  },
+  "engines": {
+    "node": "6.* || 8.* || >= 10.*"
```

Comment:
@rwjblue since 6 dies EOM should we just for 8.* ?

Reply:
I just stole this from ember-cli. I can send another PR at the end of the month to move it over to the newer node versions.